app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -75,10 +75,10 @@ class ChatRoomFragment : Fragment(), ChatRoomView {
         super.onDestroyView()
     }
 
-    override fun showMessages(dataSet: List<MessageViewModel>, serverUrl: String) {
+    override fun showMessages(dataSet: List<Message>, serverUrl: String) {
```

Comment:
This is really strange, why we are not using the MessageViewModel anymore?

Reply:
Something was wrong with the merges.. Fixing it.